qulacs-src/cppsim/circuit.hpp

```diff
@@ -1,10 +1,10 @@
 
 /**
- * @file circuit.hpp
- * 
- * @brief QuantumCircuitクラスの詳細
+ * @file gate.hpp
+ *
+ * \~japanese-en @brief 日本語説明
```

Comment:
```suggestion
 * \~japanese-en @brief QuantumCircuitクラスの詳細
```
I think this is by mistake :p